operator/src/main/java/oracle/kubernetes/operator/helpers/CallBuilder.java

```diff
@@ -594,6 +600,40 @@ public VersionInfo readVersionCode() throws ApiException {
     return DISPATCHER.execute(factory, requestParams, helper);
   }
 
+  /**
+   * Execute a synchronous call with a retry on failure.
+   * @param call The call
+   * @param retryDelaySeconds Retry delay in seconds
+   * @param <T> Call return type
+   * @return Results of operation, if successful
+   * @throws Exception Exception types other than ApiException, which will cause failure
+   */
+  public <T> T executeSynchronousCallWithRetry(Callable<T> call, int retryDelaySeconds) throws Exception {
+    T result = null;
+    boolean complete = false;
+    do {
+      try {
+        result = call.call();
+        complete = true;
+      } catch (RuntimeException re) {
+        Throwable cause = re.getCause();
+        if (cause instanceof ApiException) {
+          LOGGER.warning(MessageKeys.EXCEPTION, cause);
+        }
+      } catch (Throwable t) {
+        LOGGER.warning(MessageKeys.EXCEPTION, t);
```

Comment:
Please add a comment here that we expect the liveness probe to cancel this process if it "retries forever".  The next coder may not understand the assumption/expectation.